server/src/util/slack.ts

```diff
@@ -0,0 +1,50 @@
+import { RequisitionStatus } from "@prisma/client";
+import { WebClient } from "@slack/web-api";
+import { GraphQLError } from "graphql";
+import { projectReferenceString, requisitionReferenceString, statusToString } from "../api/resolvers/common";
+import { prisma } from "../common";
+
+const web = new WebClient(process.env.SLACK_API_TOKEN);
+
+export const sendSlackNotification = async (requisitionId: number) => {
```

Comment:
You don't have to fix this in this PR, but in the future it'd be better to create a more generic way to send notifications with any content to any user

Reply:
Yeah will do